pages/common/valgrind.md

```diff
@@ -0,0 +1,19 @@
+# valgrind
+
+> Wrapper for a set of expert tools for profiling, optimizing and debugging programs.
+
+- Test `program` with a specified Valgrind tool:
+
+`valgrind --tool={{tool_name}} {{program}}`
+
+- Use the (default) Memcheck tool to report all possible memory leaks of `program` in detail:
+
+`valgrind --leak-check=full --show-leak-kinds=all {{program}}`
```

Comment:
That'd work, yeah. Maybe this one could be the second example then. And maybe mention "with full details" rather than just "in detail" just to make the option associations clearer?